org.eclipse.xtend.m2e/lifecycle-mapping-metadata.xml

```diff
@@ -2,7 +2,7 @@
   <pluginExecutions>
     <pluginExecution>
       <pluginExecutionFilter>
-        <groupId>org.eclipse.xtend</groupId>
+        <groupId>org.eclipse.xtext</groupId>
         <artifactId>xtend-maven-plugin</artifactId>
```

Comment:
@HannesWell
i wonder if we should add two entries here, one with the old and one with the new groupId
would this work in backwards comp. way?

Reply:
> @HannesWell i wonder if we should add two entries here, one with the old and one with the new groupId would this work in backwards comp. way?

That should probably work, although I'm not absolutely sure if there is an error if the plugin is not found.
But you can check that very quickly when you implement this.